draft-ietf-cbor-7049bis.md

```diff
@@ -1357,7 +1357,7 @@ The CBOR data model for maps does not allow ascribing semantics to the
 order of the key/value pairs in the map representation.  Thus, a
 CBOR-based protocol MUST NOT specify that changing the key/value pair
 order in a map would change the semantics, except to specify that some,
-e.g. non-canonical, orders are disallowed. Timing, cache usage, and
+e.g. non-deterministic, orders are disallowed. Timing, cache usage, and
```

Comment:
Maybe write "non-deterministically encoded"?  I'm not sure we can use "non-deterministic" in all places where non-canonical is quite clear.